README.md

```diff
@@ -14,14 +14,33 @@ provider "logdna" {
   servicekey = "Your service key goes here"
 }
 
+resource "logdna_alert" "my_alert" {
+  name  = "Email and PagerDuty Alert Preset"
```

Comment:
alert preset example on top followed by view (resources also display as such in the registry- alert presets and then views)

Reply:
am splitting this into two separate examples with their own code blocks @tanberry. lmk if that looks better :) agree that the two are separate